python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -173,6 +173,8 @@ def test_library_options(arctic_client):
 
 
 def test_separation_between_libraries(arctic_client):
+    # This fails for mem-backed without the library caching implemented in
```

Comment:
I see. This is essentially the same test as my `del lib` example above. Since we are re-getting from `ac`, without caching that `Storage` object, the Arctic instance will be forced to return a new one the second time, with a fresh, blank memory. Hence the empty symbol list.